assignment-1-sql/xwalker/xavier-walker.sql

```diff
@@ -0,0 +1,43 @@
+--Question 1
+
+SELECT * FROM Fact_order
+WHERE charge_amount > 500
+AND completed_at IS NOT NULL
+ORDER BY completed_at desc
+LIMIT 1;
+
+--Question 2
+
+SELECT DATE_TRUNC('week', completed_at) AS week
+    , count(fo.completed_at) AS "Orders Completed"
+    , sum(fo.charge_amount) AS "Gross Revenue"
+FROM fact_order AS fo
+JOIN countries AS c ON c.ID = fo.country_id
+WHERE c.name = 'Canada'
+AND week BETWEEN DATEADD('week', -9, getdate()) AND DATEADD('week', -1, getdate())
+group by week
+order by week desc
+limit 10;
+
+--Question 3
+
+SELECT
+    CASE
+        WHEN DATEDIFF('month', hired_at, getdate()) <= 6 THEN '0-6'
```

Comment:
This surely will work, 
one nitpick here:
every time you see yourself copy-pasting expressions like that, it make sense to use CTE ( `with subquery as (...) select ...` thingy)

you want to put expressions there and add conditions in the main query:
```sql
with hired_drivers as ( 
  select 
    id, 
    DATEDIFF('month', hired_at, getdate()) as days_since_hired
  from drivers 
  where hired_at is not null
)
select
  case 
    when days_since_hired <= 6 then '0-6'
  end as active_months,
  count(id) / (SELECT count(id) FROM drivers WHERE hired_at IS NOT NULL) * 100 AS percentage_by_months_acitve
from hired_drivers
group by 1
```